tests/system/node/4/config.yaml

```diff
@@ -11,7 +11,6 @@ node:
   timeout: 5000
   # Path to the data directory (if the path doesn't exist it will be created)
   data_dir: .cache
-  block_interval_sec: 5
```

Comment:
Strictly speaking shouldn't this be a separate commit?

Reply:
I deserved it.